javascript/cable_ready.js

```diff
@@ -119,6 +119,24 @@ const DOMOperations = {
     dispatch(document, 'cable-ready:after-push-state', config)
   },
 
+  // Storage .................................................................................................
+
+  storageSetItem: config => {
+    const { key, value, storageType } = config
+    dispatch(document, 'cable-ready:before-storage-set-item', config)
+    storage = type === 'session' ? sessionStorage : localStorage
```

Comment:
Shouldn't this be `storageType`?

```suggestion
    storage = storageType === 'session' ? sessionStorage : localStorage
```

Or otherwhise `type` on line L125